internal/bucketserver/bucket.go

```diff
@@ -1,13 +1,13 @@
 // SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and IronCore contributors
 // SPDX-License-Identifier: Apache-2.0
 
-package server
+package bucketserver
```

Comment:
Wdyt about putting the `bucketserver` and `volumeserver` in a `server` package to keep the package name concise?